test/generate.ts

```diff
@@ -141,3 +141,20 @@ describe("The generate endpoint successfully completes with logit bias ", () =>
     expect(response.statusCode).to.equal(200);
   });
 });
+
+describe("The generate endpoint successfully completes with truncate", () => {
+  let response: cohereResponse<generateResponse>;
+  before(async function () {
+    this.timeout(5000);
+    response = await cohere.generate({
+      model: "medium",
+      prompt: "hello what is your name",
+      max_tokens: 20,
+      truncate: "END"
```

Comment:
I know that testing in sdks in general isn't the best .. but I think this test can be slightly improved, since this doesn't really assert anything (it could be that you misspelled `truncate` in the actual sdk, and it sends to the api a key=`trunc` for example which is being ignored. what do you think of something like: 
- create a long enough string .. `prompt = someString * 2048`
- request without `truncate`, and assert 400
- request with `truncate` for the same prompt, and assert a 200

Reply:
Agreed will do, i saw the other tests and fell into the "I guess it'll be resolved later" trap ;)